sentry-kotlin-multiplatform/build.gradle.kts

```diff
@@ -82,7 +82,11 @@ kotlin {
             summary = "Official Sentry SDK Kotlin Multiplatform"
             homepage = "https://github.com/getsentry/sentry-kotlin-multiplatform"
 
-            pod(Config.Libs.sentryCocoa, "~> ${Config.Libs.sentryCocoaVersion}")
+            pod(Config.Libs.sentryCocoa) {
+                source = git("https://github.com/getsentry/sentry-cocoa.git") {
+                    tag = Config.Libs.sentryCocoaVersion
+                }
+            }
```

Comment:
Why are we changing the source to `GH`?

Reply:
Installing through cocoapods directly did not resolve for 8.3.0 (not sure how in sync they are with gh releases) and I was trying to keep it close to [sentry docs](https://docs.sentry.io/clients/cocoa/) since they also use gh

Reply:
it should be something else, Cocoapods and GH should have the same version, for every release.
https://cocoapods.org/pods/Sentry#changelog
Let's focus on releasing the current version instead of fixing this now.

Reply:
turned out to be a local gradle cache error so using cocoapods still works as expected